.asf.yaml

```diff
@@ -18,6 +18,18 @@ github:
     merge:  false
     rebase: false
 
+  protected_branches:
+    main: {}
```

Comment:
Is there a way to add `branch_*`?

Reply:
I cannot see it documented. But we could easily enumerate all of them and let the release wizard add new branches to the list?

Reply:
Also, `.asf.yaml` does not support glob format (see https://the-asf.slack.com/archives/CBX4TSBQ8/p1668009454564229?thread_ts=1667901002.937809&cid=CBX4TSBQ8), but will probably be added once GH api supports it.